src/Adapter/PlatformServices.Desktop/Services/DesktopTestSourceHost.cs

```diff
@@ -359,6 +359,7 @@ private void AppDomainCreationDisabledInRunSettings()
         if (this.runSettings != null && MSTestAdapterSettings.IsAppDomainCreationDisabled(this.runSettings.SettingsXml))
         {
                 this.isAppDomainCreationDisabled = true;
+                return;
```

Comment:
Change the function name to `IsAppDomainCreationDisabledInRunSettings`  make return type `bool`. or assign value of `this.runSettings != null && MSTestAdapterSettings.IsAppDomainCreationDisabled(this.runSettings.SettingsXml)` to isAppDomainCreationDisabled in .ctor and remove this function.